packages/fury-adapter-oas3-parser/lib/parser/oas/parseComponentsObject.js

```diff
@@ -31,9 +31,17 @@ function parseComponentsObject(context, element) {
   const validateIsObject = key => R.unless(isObject,
     createWarning(namespace, `'${name}' '${key}' is not an object`));
 
+  const parseSchemaMember = member => pipeParseResult(namespace,
+    R.compose(parseSchemaObject(context), getValue),
+    (dataStructure) => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
I understand what this eslint rule is meant to protect against and in a lot of cases I agree with it. However, in this particular line I think this makes sense, we're modifying the dataStructure to attach the ID.